services/federator/test/unit/Test/Federator/ExternalServer.hs

```diff
@@ -72,6 +73,19 @@ requestBrigFailure =
       embed $ assertEqual "one call to brig should be made" [expectedCall] actualCalls
       embed $ assertEqual "response should be success with correct body" (InwardResponseErr "Invalid HTTP status from component: 404 Not Found") res
 
+requestGalleySuccess :: TestTree
+requestGalleySuccess =
+  testCase "should translate response from galley to 'InwardResponseBody' when response has status 200" $
+    runM . evalMock @Service @IO $ do
+      mockServiceCallReturns @IO (\_ _ _ _ -> pure (HTTP.ok200, Just "response body"))
+      let request = Request Galley "/users" "\"foo\"" exampleDomain
+
+      res :: InwardResponse <- mock @Service @IO . noLogs . Polysemy.runReader allowAllSettings $ callLocal request
+      actualCalls <- mockServiceCallCalls @IO
+      let expectedCall = (Galley, "/users", "\"foo\"", aValidDomain)
```

Comment:
I don't understand this well, but shouldn't this be about a Galley-related endpoint?

Reply:
I guess it should. It's just a mocked request, so it doesn't matter, but I'll turn it into a galley endpoint for clarity.